diff --git a/control-plane/pkg/reconciler/broker/namespaced_broker.go b/control-plane/pkg/reconciler/broker/namespaced_broker.go
index 8745fada7..8612d3846 100644
--- a/control-plane/pkg/reconciler/broker/namespaced_broker.go
+++ b/control-plane/pkg/reconciler/broker/namespaced_broker.go
@@ -412,6 +412,7 @@ func (r *NamespacedReconciler) configMapsFromSystemNamespace(broker *eventing.Br
 		"config-observability",
 		"config-features",
 		"kafka-config-logging",
+		"config-openshift-trusted-cabundle",
 	}
 	resources := make([]unstructured.Unstructured, 0, len(configMaps))
 	for _, name := range configMaps {
