diff --git a/control-plane/pkg/reconciler/broker/namespaced_broker.go b/control-plane/pkg/reconciler/broker/namespaced_broker.go
index 5e71fa6d0..4db94dede 100644
--- a/control-plane/pkg/reconciler/broker/namespaced_broker.go
+++ b/control-plane/pkg/reconciler/broker/namespaced_broker.go
@@ -386,6 +386,7 @@ func (r *NamespacedReconciler) configMapsFromSystemNamespace(broker *eventing.Br
 		"config-kafka-broker-data-plane",
 		"config-tracing",
 		"kafka-config-logging",
+		"config-openshift-trusted-cabundle",
 	}
 	resources := make([]unstructured.Unstructured, 0, len(configMaps))
 	for _, name := range configMaps {
