diff --git a/data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleBuilder.java b/data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleBuilder.java
index f9053ea2d..c2d7b1813 100644
--- a/data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleBuilder.java
+++ b/data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/ConsumerVerticleBuilder.java
@@ -213,7 +213,7 @@ public class ConsumerVerticleBuilder {
     }
 
     private WebClientOptions createWebClientOptionsFromCACerts(final String caCerts) {
-        final var pemTrustOptions = new PemTrustOptions();
+        final var pemTrustOptions = new PemTrustOptions(openshiftPemTrustOptions());
         for (String trustBundle : consumerVerticleContext.getTrustBundles()) {
             pemTrustOptions.addCertValue(Buffer.buffer(trustBundle));
         }
@@ -223,6 +223,11 @@ public class ConsumerVerticleBuilder {
         return new WebClientOptions(consumerVerticleContext.getWebClientOptions()).setTrustOptions(pemTrustOptions);
     }
 
+  private PemTrustOptions openshiftPemTrustOptions() {
+    // TODO: Go for all files
+    return new PemTrustOptions().addCertPath("/ocp-serverless-custom-certs/ca-bundle.crt");
+  }
+
     private ResponseHandler createResponseHandler(final Vertx vertx) {
         if (consumerVerticleContext.getEgress().hasReplyUrl()) {
             return new ResponseToHttpEndpointHandler(new WebClientCloudEventSender(
