diff --git a/test/e2e_new/broker_test.go b/test/e2e_new/broker_test.go
index 52f168a9f..1522c7ef8 100644
--- a/test/e2e_new/broker_test.go
+++ b/test/e2e_new/broker_test.go
@@ -117,6 +117,8 @@ func TestBrokerCannotReachKafkaCluster(t *testing.T) {
 }
 
 func TestNamespacedBrokerResourcesPropagation(t *testing.T) {
+	t.Skip("We propagate service monitors in Serverless Operator so this test won't work for now")
+
 	ctx, env := global.Environment(
 		knative.WithKnativeNamespace(system.Namespace()),
 		knative.WithLoggingConfig,
